writeit/urls.py

```diff
@@ -28,6 +30,7 @@
 
     # TODO: These can probably be removed at some point.
     url(r'^contactos/', include('contactos.urls')),
+    url(r'^incoming_mail/$', csrf_exempt(IncomingMail.as_view()), name='incoming_mail'),
```

Comment:
```suggestion
    url(r'^sendgrid_inbound_webhook/$', csrf_exempt(SendgridInboundWebhook.as_view()), name='sendgrid_inbound_webhook'),
```

What do you think of this? While I hate vendor specifics in a URL, that's exactly what this is, so let's make it very clear.

Reply:
@jbothma Yeah, I don't mind either way... but really, any server could send mail to this endpoint. In future might want to use it in it's most general case. Then a refactor would be needed. What is the value of this change?

Reply:
We test the endpoint for example, in an agnostic way... so theoretically it doesn't need to be coming from sendgrid only. What is received is also nothing specific to Sendgrid either.

Reply:
the sendgrid-specific part is the that posts a raw MIME blob in the mail field of a multipart encoded body

Not sure if these options even support roaw bodies, but their parsed bodies all differ
https://documentation.mailgun.com/en/latest/quickstart-receiving.html#examples-of-filter-expressions-for-routes
https://dev.mailjet.com/email/guides/parse-api/

So I'm not sure if anyone else would actually send a request that this would understand.